datamap/mv/core/src/main/scala/org/apache/carbondata/mv/datamap/MVDataMapProvider.scala

```diff
@@ -55,6 +68,10 @@ class MVDataMapProvider(
   }
 
   override def initData(): Unit = {
+    if (!dataMapSchema.isLazy) {
+      rebuild()
+      DataMapStatusManager.enableDataMap(dataMapSchema.getDataMapName)
```

Comment:
During enabling of MV datamap we should check the mapping and make sure the parent and child datamaps are completely synchronized.